recipes/centos_spring_boot/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+set -e
+
+# insert settings for mirrors/repository managers into settings.xml if supplied
+function configure_mirrors {
```

Comment:
only `configure_mirrors() {` ?

Reply:
yes thats it for now  - we might need a PROXY config as well. but am not adding that to this as I can have separate/issue/pr for it

Reply:
I only mean : replace
```
 function abcd { 
```
by
```
abcd() {
```

Reply:
yeah got it done after using the shellcheck tool . thanks for clarification